.github/workflows/spelling2.yml

```diff
@@ -91,86 +91,93 @@ jobs:
-        experimental_apply_changes_via_bot: ${{ github.repository_owner != 'microsoft' && 1 }}
-        use_sarif: ${{ (!github.event.pull_request || (github.event.pull_request.head.repo.full_name == github.repository)) && 1 }}
-        extra_dictionary_limit: 20
-        extra_dictionaries:
-          cspell:software-terms/dict/softwareTerms.txt
-          cspell:cpp/src/stdlib-cpp.txt
-          cspell:lorem-ipsum/dictionary.txt
-          cspell:cpp/src/stdlib-c.txt
-          cspell:php/dict/php.txt
-          cspell:filetypes/filetypes.txt
-          cspell:java/src/java.txt
-          cspell:python/src/common/extra.txt
-          cspell:node/dict/node.txt
-          cspell:java/src/java-terms.txt
-          cspell:aws/aws.txt
-          cspell:typescript/dict/typescript.txt
-          cspell:dotnet/dict/dotnet.txt
-          cspell:golang/dict/go.txt
-          cspell:fullstack/dict/fullstack.txt
-          cspell:cpp/src/compiler-msvc.txt
-          cspell:python/src/python/python-lib.txt
-          cspell:mnemonics/src/mnemonics.txt
-          cspell:cpp/src/stdlib-cmath.txt
-          cspell:css/dict/css.txt
-          cspell:cpp/src/lang-keywords.txt
-          cspell:django/dict/django.txt
-          cspell:python/src/python/python.txt
-          cspell:html/dict/html.txt
-          cspell:cpp/src/ecosystem.txt
-          cspell:cpp/src/compiler-clang-attributes.txt
-          cspell:npm/dict/npm.txt
-          cspell:r/src/r.txt
-          cspell:powershell/dict/powershell.txt
-          cspell:csharp/csharp.txt
+      - name: check-spelling
+        id: spelling
+        uses: check-spelling/check-spelling@v0.0.24
+        with:
+          suppress_push_for_open_pull_request: ${{ github.actor != 'dependabot[bot]' && 1 }}
+          checkout: true
+          check_file_names: 1
+          spell_check_this: microsoft/terminal@main
+          post_comment: 0
+          use_magic_file: 1
+          report-timing: 1
+          warnings: bad-regex,binary-file,deprecated-feature,ignored-expect-variant,large-file,limited-references,no-newline-at-eof,noisy-file,non-alpha-in-dictionary,token-is-substring,unexpected-line-ending,whitespace-in-dictionary,minified-file,unsupported-configuration,no-files-to-check,unclosed-block-ignore-begin,unclosed-block-ignore-end
+          experimental_apply_changes_via_bot: ${{ github.repository_owner != 'microsoft' && 1 }}
+          use_sarif: ${{ (!github.event.pull_request || (github.event.pull_request.head.repo.full_name == github.repository)) && 1 }}
+          check_extra_dictionaries: ""
+          dictionary_source_prefixes: >
+            {
+            "cspell": "https://raw.githubusercontent.com/check-spelling/cspell-dicts/v20241114/dictionaries/"
```

Comment:
I'm in the process of updating the `cspell` dictionaries, so this pulls in what I expect to be the next version with their refreshed content and my refreshed layout. I expect it'll be included in v0.0.25.